qa-heal.planx-pla.net/metadata/aggregate_config.json

```diff
@@ -1,1240 +1,1296 @@
 {
-    "gen3_commons": {
-        "HEAL": {
-            "mds_url": "https://qa-heal.planx-pla.net/",
-            "commons_url": "https://qa-heal.planx-pla.net/"
+  "configuration": {
+    "schema": {
+      "__manifest": {
+        "type": "array",
+        "properties": {
+          "file_name": {
+            "type": "string"
+          },
+          "file_size": {
+            "type": "integer"
+          }
+        }
+      },
+      "commons_url": {},
+      "tags": {
+        "type": "array"
+      },
+      "advSearchFilters": {
+        "type": "array",
+        "default": []
+      },
+      "study_description_summary": {},
+      "project_title": {}
+    },
+    "settings": {
+      "cache_drs": true
+    }
+  },
+  "adapter_commons": {
+    "HEAL": {
+      "mds_url": "https://qa-heal.planx-pla.net/",
+      "commons_url": "https://qa-heal.planx-pla.net/",
+      "adapter": "gen3",
+      "config" : {
+        "guid_type": "unregistered_discovery_metadata",
+        "study_field": "gen3_discovery"
+      },
+      "keep_original_fields": true,
+      "field_mappings": {
+        "authz": "path:authz",
+        "tags": { "path":"tags",
+          "default": [] },
+        "__manifest": { "path":"__manifest",
+          "default": [] },
+        "advSearchFilters": {
+          "path":"advSearchFilters"
+        },
+        "study_description_summary": "path:study_description_summary",
+        "manifest": "path:__manifest",
+        "project_title": "path:project_title",
+        "commons_url": "https://qa-heal.planx-pla.net/"
+      }
+    },
+    "SAMHDA": {
+      "mds_url": "https://qa-heal.planx-pla.net/",
+      "commons_url": "https://qa-heal.planx-pla.net",
+      "adapter": "gen3",
+      "keep_original_fields": false,
+      "select_field": {
+        "field_name": "data_resource",
+        "field_value": "SAMHDA"
+      },
```

Comment:
what does the select_field do?

Reply:
Tested this new config in my dev env, it errors out not sure what I'm missing happy to sync later:
```
[2022-08-23 19:43:02,435][       mds][   INFO] Received 835 from HEAL
Traceback (most recent call last):
  File "/src/src/mds/populate.py", line 187, in <module>
    asyncio.run(main(commons_config=commons, hostname=args.hostname, port=args.port))
  File "/usr/local/lib/python3.9/asyncio/runners.py", line 44, in run
    return loop.run_until_complete(main)
  File "/usr/local/lib/python3.9/asyncio/base_events.py", line 642, in run_until_complete
    return future.result()
  File "/src/src/mds/populate.py", line 135, in main
    await populate_metadata(name, common, results)
  File "/src/src/mds/populate.py", line 84, in populate_metadata
    await datastore.update_metadata(
  File "/src/src/mds/agg_mds/datastore/__init__.py", line 32, in update_metadata
    await client.update_metadata(*args)
  File "/src/src/mds/agg_mds/datastore/elasticsearch_dao.py", line 134, in update_metadata
    elastic_search_client.index(
  File "/env/lib/python3.9/site-packages/elasticsearch/client/utils.py", line 101, in _wrapped
    return func(*args, params=params, **kwargs)
  File "/env/lib/python3.9/site-packages/elasticsearch/client/__init__.py", line 370, in index
    return self.transport.perform_request(
  File "/env/lib/python3.9/site-packages/elasticsearch/transport.py", line 402, in perform_request
    status, headers_response, data = connection.perform_request(
  File "/env/lib/python3.9/site-packages/elasticsearch/connection/http_urllib3.py", line 252, in perform_request
    self._raise_error(response.status, raw_data)
  File "/env/lib/python3.9/site-packages/elasticsearch/connection/base.py", line 253, in _raise_error
    raise HTTP_EXCEPTIONS.get(status_code, TransportError)(
elasticsearch.exceptions.RequestError: RequestError(400, 'mapper_parsing_exception', 'object mapping for [tags] tried to parse field [tags] as object, but found a concrete value')
command terminated with exit code 1
```